docs/src/tutorials/standalone/Usage/domain_tutorial.jl

```diff
@@ -74,7 +74,7 @@ As discussed above, our modeling requires that variables of a model can be defin
 
 There is a single key method which take a ClimaLand domain as an argument.
 
-- [` coordinates(domain)`](https://clima.github.io/ClimaLand.jl/stable/APIs/shared_utilities/#ClimaLand.Domains.coordinates): under the hood, this function  uses
+- [` coordinates(domain)`](@ref ClimaLand.Domains.coordinates): under the hood, this function  uses
```

Comment:
Later in this file, could we change this line?
"a Point domain only has a coordinate z_sfc (surface only). Column domains have a surface coordinate of z_sfc" -> "a Point domain only has a coordinate z\_sfc (surface only). Column domains have a surface coordinate of z\_sfc"?

I noticed that in the rendered page, it is interpreting the underscores as indicating italics

Reply:
Likewise for "The user still must define the prognostic\_domain\_names method"